core/src/main/scala/com/clairvoyant/restonomer/core/authentication/RestonomerAuthentication.scala

```diff
@@ -54,6 +58,39 @@ case class BasicAuthentication(
 
 }
 
+case class JwtAuthentication(
+    subject: String,
+    secretKey: String,
+    algorithm: String = JwtAlgorithm.HS256.name,
+    tokenExpiresIn: Long = 1800
+) extends RestonomerAuthentication {
+
+  implicit val clock: Clock = Clock.systemUTC
+  var encodingAlgo: JwtAlgorithm = _
```

Comment:
encodingAlgo is not used anywhere. Kindly remove.

Reply:
done